easybuild/easyblocks/c/cuda.py

```diff
@@ -114,3 +148,12 @@ def make_module_req_guess(self):
         })
 
         return guesses
+    
+    def make_module_extra(self):
+        """Specify CUDA_HOME """
+
+        txt = super(EB_CUDA, self).make_module_extra()
+
+        txt += self.module_generator.set_environment('CUDA_HOME', self.installdir)
```

Comment:
OK, I traced not having `$CUDA_HOME` or `$CUDA_PATH` being defined down to a bug in the framework.
It has been there for a while (March 2014), but luckily only affects CUDA.

I'll get that fixed as a last-minute thing in the framework, but this PR needs a bit more love before it can go in, so it'll have to be post v2.5.0.